profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/tmgmt_poetry_mock.behat.inc

```diff
@@ -195,4 +195,26 @@ class TMGMTPoetryMockSubContext extends RawDrupalContext implements DrupalSubCon
     }
   }
 
+  /**
+   * Verifies that poetry received initial request with NEXT_EUROPA_COUNTER.
+   *
+   * @Then the poetry translation service received initial request with automatic counter
+   */
+  public function assertPoetryTranslationServiceReceivedTheInitialRequest() {
+    $message = '';
+    if (is_object($this->translationRequest['file'])) {
+      $file_object = $this->translationRequest['file'];
+      $message = file_get_contents($file_object->uri);
+    }
+
+    if (strpos($message, PoetryMock::COUNTER_STRING) === FALSE) {
```

Comment:
This check is a bit 'wide'. Would it be feasible to look for the exact element?
The fact is, this test probably succeeds without problems without the fix, because we were sending the sequence value inside the "numero" element while it should be in the "sequence" element. At first sight this code will not catch this mistake.